classes/Currency.php

```diff
@@ -379,6 +434,20 @@ public function getName()
         return Tools::ucfirst($this->name[$id_lang]);
     }
 
+    public function getSymbol()
+    {
+        if (is_string($this->symbol)) {
+            return $this->symbol;
+        }
+
+        $id_lang = $this->id_lang;
```

Comment:
Please use the Null Coalescing Operator
```suggestion
        $id_lang = $this->id_lang ?? Configuration::get('PS_LANG_DEFAULT');
```

Reply:
No because we must be compatible with php5.6 and this one has been added in 7.0 https://www.php.net/manual/en/migration70.new-features.php